src/main/osd/osd_elements.c

```diff
@@ -992,6 +993,21 @@ static void osdElementFlymode(osdElementParms_t *element)
     }
 }
 
+static void osdElementReadyMode(osdElementParms_t *element)
+{
+    if (IS_RC_MODE_ACTIVE(BOXREADY) && !isFlying && calculateThrottlePercent() < 1) {
+            strcpy(element->buff, "READY"); 
+    }
```

Comment:
Should have an else clause here to ensure the `element->buff` is always valid even if only a NULL string.

Reply:
element->buff is already cleared before calling this function

Reply:
Ah, right. 👍🏻